scio-spanner/src/main/scala/com/spotify/scio/spanner/package.scala

```diff
@@ -22,7 +22,7 @@ import com.spotify.scio.coders.Coder
 import com.spotify.scio.io.Tap
 import com.spotify.scio.values.SCollection
 import org.apache.beam.sdk.{coders => bcoders}
-import org.apache.beam.sdk.io.gcp.spanner.{ReadOperation, SpannerConfig}
+import org.apache.beam.sdk.io.gcp.spanner.{MutationGroup, ReadOperation, SpannerConfig}
```

Comment:
wdyt about adding an implicit coder for [Transaction](https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/Transaction.java), too? It's the only other `Serializable` class left in Beam's Spanner package that we're not including here :)

Reply:
I was thinking about this and I'm not sure that it would be used. The only way to supply `Transaction` is through a `PCollectionView` and i don't think the users should be moving around `Transaction`'s because the only way to create one is to actually connect to spanner and get one.